userscripts/commit-ref.user.js

```diff
@@ -11,6 +12,18 @@
 /*
  * Copyright (C) 2023 Andrei Rybak
  *
+ * This program is free software: you can redistribute it and/or modify
```

Comment:
It'd probably make more sense to have a top-level LICENSES directory (like Linux https://github.com/torvalds/linux) and then add SPDX to every file. Until I find the time to do that, I suppose this is fine.